src/test/kotlin/org/rust/ide/intentions/NestUseStatementsIntentionTest.kt

```diff
@@ -335,4 +335,129 @@ class NestUseStatementsIntentionTest : RsIntentionTestBase(NestUseStatementsInte
         };
 
     """)
+
+    fun `test renamed import 1`() = doAvailableTest("""
+        pub mod foo {
+            pub fn bar() {}
+            pub fn quux() {}
+        }
```

Comment:
Existing tests don't contain declarations and the intention is purely syntactic, so probably we can remove it